docs/site/Context.md

```diff
@@ -258,6 +258,27 @@ emitted for the existing binding followed by a `bind` event for the new binding.
 If a context has a parent, binding events from the parent are re-emitted on the
```

Comment:
The list above says that the `Context` emits `bind`, `unbind` and `error` events. I think we need to add `changed` event to that list?